docs/index.rst

```diff
@@ -87,9 +87,9 @@ Example notebooks
 
    Contributor Guide <contributing/index>
    contributing/many_ways
-   contributing/getting_ready
+   Getting Ready to Contribute <contributing/getting_ready>
```

Comment:
Emojis in titles in the sidebar appear weird on some browsers (i.e., showing up only the left side of the emoji).